website/source/docs/guides/managing-acl-policies.html.md

```diff
@@ -0,0 +1,150 @@
+
+```hcl
+agent_prefix "" 
+  { policy = "read"
+  }
+```
+
+## Secure Access Control: Operator-Only Access
+
+The most secure access control implementation restricts tokens with `acl=”write”` policies to only one or a few trusted operators. Tokens with the policy `acl = "write"` grant the holder unlimited privileges, because they can generate tokens with any other resource and policy. The operators are responsible for creating all other policies and tokens that grant limited access to the datacenter. We refer to this implementation as the operator-only implementation. This implementation type is the most secure, and most complex to manage.  
+
+For this implementation type operators are responsible for managing policies and tokens for: 
+- service registration
+- Connect proxy registration
+- intention management
+- agent management
+- API, CLI, and UI access
+
+For static or non-containerized workflows, this implementation type is straightforward and the operator's workload scales linearly. If the workflow is dynamic, implementing an automation tool will ensure the operator's workload does not scale exponentially. 
+
+!> If you need to share token generation responsibilities with other teams, anyone with the ability to create tokens (`acl = "write"`) will also have unrestricted access to the datacenter. That individual will have the ability to deregister agents and services, and delete data from Consul KV. 
+
+### Operator-Only Implementation Example 
+
+In this following example Operators retain responsibility for service token management, but delegate access control between Connect-enabled services to the security team. 
+
+-> Note: The service registration examples describe the token generation process for per-service tokens, therefore, they are only applicable if you are able to create per-service tokens. 
+
+Initially, the Operator creates a single token with intention management privileges for the Security Team, and another service token for the Developer. For intention management the `intention` policy disposition should be included in the service rule.
+
+```hcl
+service "wordpress" 
+  { policy = "read" 
+    intentions = "write"
+  }
+```
+
+This enables the security team to create an intention that allows the upstream proxy for `mysql` to send data to the destination, the downstream proxy for `wordpress`.
+
+```sh
+consul intention create wordpress mysql
+``` 
+
+The Security Team responsible for managing intentions would need to create allow intentions when the `default_policy` is set to `deny`, since deny all will be inherited from the ACL configuration.
+
+In this implementation type, Developers are responsible for requesting a token for their service. For a Connect enabled service, the Operator would need to create a policy that provides write privileges for the service and proxy, and read privileges for the service_prefix and node_prefix.
```

Comment:
```suggestion
In this implementation type, Developers are responsible for requesting a token for their service. For a Connect enabled service, the Operator would need to create a policy that provides write privileges for the service and proxy, and read privileges for all services and nodes to enable discovery of other upstream dependencies.
```

Maybe worth adding a note here that it's possible to be more restrictive with service and node read privs but it quickly adds complexity to whitelist all the services and nodes they run on that a token should be allowed to discover upstreams via DNS, API or Connect.